.jshintrc

```diff
@@ -0,0 +1,15 @@
+{
+	"boss": true,
+	"curly": true,
+	"eqeqeq": true,
+	"eqnull": true,
+	"expr": true,
+	"immed": true,
+	"noarg": true,
+	"quotmark": "double",
```

Comment:
That's something we can fight out later, but JSCS already does code style linting. Should we keep this rule in here then? This is how it's in the style guide so maybe that's more of a discussion for there.